### ISSUE TITLE
 <svg> element appended to end of body should be invisible to screen readers

### ISSUE THREAD

**Issue by lenzls** [svgdotjs/svg.js#1023]
Similar to #908, the svg element still distracts screen readers.

It should have the `aria-hidden="true"` [attribute](https://www.w3.org/TR/wai-aria-1.1/#aria-hidden) set.

**Comment by Fuzzyma** [comment]
Feel free to create a PR to fix this issue! :)

**Comment by lenzls** [comment]
added one :)
